app/controllers/submit.js

```diff
@@ -1433,11 +1451,11 @@ export default Controller.extend(Analytics, BasicsValidations, COIValidations, N
     },
 
     _moveFromBasics() {
-        this.send('next', this.get('_names.3'));
+        this.send('next', 'Basics');
```

Comment:
Because of the somewhat dynamic `_names` array, instead of referring to panel name using array index, we prefer using actual strings.